packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -41,9 +42,15 @@ const NavLogo = ({
   link,
   ...customProps
 }) => {
+  const imgProps = {
+    className: cx('img'),
+  };
+  if (url) {
```

Comment:
you can condense this into a neat little es6 operation:
```
const imgProps = {
  className: cx('img'),
  ...url && { src: url },
};
```

Reply:
[updated](https://github.com/cerner/terra-consumer/pull/60/files)